README.adoc

```diff
@@ -61,21 +60,14 @@ From now on, this README assumes you're running commands from the `riff` reposit
 cd riff
 ----
 
-==== Build the kubernetes-model jar Enhanced with the Types for the CustomResources
-
-[source, bash]
-----
-./build-model upstream riff
-----
-
 ==== Point your local Docker environment to the Docker environment running in minikube:
 
 [source, bash]
 ----
 eval $(minikube docker-env)
 ----
 
-==== Build the Function Sidecar, HTTP Gateway and Topic Controller
+==== Build the Function Sidecar, HTTP Gateway, and Topic Controller
```

Comment:
Couldn't we add the build of the Function Controller here as well, looks weird to have that as a separate step later now that it's a Go project

Reply:
it was originally due to the tests expecting those resources to be created, but if that's no longer a requirement, I agree they should all be together

Reply:
I'll submit a separate PR with some other changes as well